crates/aptos-push-metrics/src/lib.rs

```diff
@@ -86,20 +116,71 @@ impl MetricsPusher {
             "Starting push metrics loop. Sending metrics to {} with a frequency of {} seconds",
             push_metrics_endpoint, push_metrics_frequency_secs
         );
+
+        info!(
+            "Execution dashboard link: {} ",
```

Comment:
The MetricPusher is used by other tooling as well, it's confusing to print this here.  Can you move these out of the pusher?